Composer/packages/lib/code-editor/src/hooks/useSelection.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ISelection, Selection, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import * as React from 'react';
+
+export const useSelection = <T>(selectionMode?: SelectionMode) => {
+  const [items, setItems] = React.useState<T[]>([]);
+
+  const selection = React.useRef<ISelection>();
+
+  if (!selection.current) {
```

Comment:
Should this if-statement be in a `useEffect` hook? I guess functionally it wouldn't make a difference, but we would have to check if `selection.current` is falsy on every render.

Reply:
actually forgot to remove it, it had other issues.